packages/frontend/src/styles/main.css

```
@@ -0,0 +1,478 @@
+  height: 20px;
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  color: var(--color-text);
+}
+
+.hamburger__bar {
+  display: block;
+  height: 2px;
+  background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)